core/extension/gdnative_interface.cpp

```diff
@@ -678,6 +681,59 @@ static const char32_t *gdnative_string_operator_index_const(const GDNativeString
 	return &self->ptr()[p_index];
 }
 
+static void gdnative_string_operator_plus_eq_string(GDNativeStringPtr p_self, const GDNativeStringPtr p_b) {
+	String *self = (String *)p_self;
+	const String *b = (const String *)p_b;
+	*self += *b;
+}
+
+static void gdnative_string_operator_plus_eq_char(GDNativeStringPtr p_self, char32_t p_b) {
+	String *self = (String *)p_self;
+	*self += p_b;
+}
+
+static void gdnative_string_operator_plus_eq_cstr(GDNativeStringPtr p_self, const char *p_b) {
+	String *self = (String *)p_self;
+	*self += p_b;
+}
+
+static void gdnative_string_operator_plus_eq_wcstr(GDNativeStringPtr p_self, const wchar_t *p_b) {
+	String *self = (String *)p_self;
+	*self += p_b;
+}
+
+static void gdnative_string_operator_plus_eq_c32str(GDNativeStringPtr p_self, const char32_t *p_b) {
+	String *self = (String *)p_self;
+	*self += p_b;
+}
+
+static GDNativeInt gdnative_xml_parser_open_buffer(GDNativeObjectPtr p_instance, const uint8_t *p_buffer, size_t p_size) {
```

Comment:
Why is it needed to have those functions here instead of putting them among the regular helpers functions that are accessible to GDScript ?

Reply:
AFAIK, binding GDScript function with pointer arguments is not supported (it only works for GDExtension specific virtual methods).

And usual `PackedByteArray` wrappers involve making a copy of data at least once, with something like SVG in OT, which can be accessing 15 MB large XML that is already in memory (loaded by third party library) it's too wasteful.